tests/test_config_models.py

```diff
@@ -5,20 +5,17 @@
 
 import ddt
 from django.contrib.auth.models import User
-from django.test import TestCase
 from django.utils import six
 from rest_framework.test import APIRequestFactory
 
 from freezegun import freeze_time
 
-from mock import patch, Mock
```

Comment:
It looks like this was the only usage of `mock` in the package, you should be able to drop it from `requirements/test.in` now.  And if we need it again later, we could import it as `unittest.mock` without requiring the extra package.